packages/block-editor/src/components/link-control/style.scss

```diff
@@ -191,6 +191,9 @@ $preview-image-height: 140px;
 		overflow-wrap: break-word;
 
 		.block-editor-link-control__search-item-info {
+			color: $gray-700;
```

Comment:
This does meet AA level for normal text, but fails at AAA. It is pretty hard to read at this color at such a small text size. Could we increase the contrast here?

Reply:
AA is the requirement. I did base this off size/color from existing UI, to not introduce additional design patterns: 

![CleanShot 2024-01-23 at 11 58 55](https://github.com/WordPress/gutenberg/assets/1813435/a856c197-3711-4698-9833-f9548ef64dc0)

![CleanShot 2024-01-23 at 12 00 20](https://github.com/WordPress/gutenberg/assets/1813435/74c2be93-3267-4c27-b469-f18b29f01fbd)